scripts/ec2-stage

```diff
@@ -158,7 +184,9 @@ cd amazon-ecs-agent
 tar -xvz -f "${ec2_work_root}/source.tgz"
 git checkout "${commit}"
 make get-deps
-scripts/stage -b ${ARTIFACT_BUCKET} -d false
+for target_os in ${TARGET_OS_LIST[@]}; do
```

Comment:
(github won't let me comment on line 176, but that's really where this is targeted) Can you install `zip` and `unzip` explicitly as well?